tests/src/test/scala/cats/tests/NonEmptyCollectionSuite.scala

```diff
@@ -153,6 +153,12 @@ abstract class NonEmptyCollectionSuite[U[+_], NE[+_], NEC[x] <: NonEmptyCollecti
     }
   }
 
+  test("grouped is consistent with iterator.toList.grouped") {
+    forAll { (is: NE[Int], i: Int) =>
+      (i > 0) ==> assert(is.grouped(i).toList.map(toList) === is.iterator.toList.grouped(i).toList)
```

Comment:
The above snippet gives a type error, but this should fix it

```suggestion
      (i > 0) ==> is.grouped(i).toList.map(toList) === is.iterator.toList.grouped(i).toList
```